### PR TITLE
Feat/load client

### PR DESCRIPTION
**Pull request by katerinamartysko**
Fetch client's details by ID

### CODE REVIEW

**Comment by katerinamartysko** [comment]
[Client list](https://trello.com/c/nL8U9tlm/7-client-list)

File: src/components/ClientsId.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+export const ClientsId: FC<Props> = ({ clientID }) => {
+  const dispatch = useDispatch();
+
+  const [fetchClientsById] = useFetching(async (id: number) => {
+    const response = await PostService.getClientId(id);
+    dispatch(getClientId(response.data));
+  });
+
+  useEffect(() => {
+    if (clientID === null) return;
```
Comment (LexaLukovka) [inline src/components/ClientsId.tsx]: Create a separate variable for  `clientID === null`. Like this
`const isNoClientID = clientID === null`

**Comment by katerinamartysko** [comment]
[Fetch client's details by ID](https://trello.com/c/leak3pTd/12-fetch-clients-details-by-id)

File: src/components/ClientsId.tsx (hunk 1 above)
Comment (katerinamartysko) [reply src/components/ClientsId.tsx]: Done

**Review by LexaLukovka — APPROVED** [review]

**Pull request merged**